src/Menu/Menu.stories.tsx

```diff
@@ -7,11 +7,80 @@ import styled, { ThemeProvider } from '@xstyled/styled-components';
 // COMPONENTS
 import { Menu } from './Menu.component';
 import { Item } from './Item';
-import { ChevronDown } from '../Icon';
+import { ChevronDown, ChevronUp } from '../Icon';
 // README
 import * as README from './README.md';
 import { RootTheme } from '../theme';
 import { Typography } from '../Typography';
+import { DropDown } from '../DropDown';
+import { List, Item as ListItem } from '../List';
+
+const listItemArray = [
+    { key: 1, label: 'List Item 1' },
+    { key: 2, label: 'List Item 2' },
+    { key: 3, label: 'List Item 3' },
+    { key: 4, label: 'List Item 4' },
+    { key: 5, label: 'List Item 5' },
+];
+
+const MockList = () => (
+    <List>
+        {listItemArray.map(({ key, label }: any) => (
+            <ListItem
+                key={key}
+                onClick={() => {
+                    /* tslint:disable no-console */
+                    console.log(`${label} has been clicked`);
+                    /* tslint:enable no-console */
+                }}
+            >
+                {label}
+            </ListItem>
+        ))}
+    </List>
+);
+
+const renderDemoMenu = () => {
+    const [isItemHovered, setIsItemHovered] = React.useState<boolean>(false);
+    const [isItem2Hovered, setIsItem2Hovered] = React.useState<boolean>(false);
+    return (
+        <Menu>
+            <DropDown
+                overlay={<MockList />}
+                position="bottomStart"
+                showArrow={false}
+                trigger="click"
+                onTriggered={setIsItemHovered}
```

Comment:
We could also solve this with [render props](https://reactjs.org/docs/render-props.html). Not sure if we actually should, but food for thought.